object_detection/mlcube/workspace/parameters.yaml

```diff
@@ -0,0 +1,5 @@
+SAVE_CHECKPOINTS: "True" # Instead of False use empty value
```

Comment:
SAVE_CHECKPOINTS should be False by default since that code path is not well tested in the recent past.

Reply:
Fixed.